packages/aws-cdk-lib/aws-events-targets/lib/log-group-resource-policy.ts

```diff
@@ -15,6 +15,10 @@ export interface LogGroupResourcePolicyProps {
    * The policy statements for the log group resource logs
    */
   readonly policyStatements: [iam.PolicyStatement];
+  /**
+   * Whether to install latest AWS SDK for the custom resource
+   */
+  readonly installLatestAwsSdk?: boolean;
```

Comment:
```suggestion
  /**
   * Whether to install latest AWS SDK for the custom resource
   * @default - install latest AWS SDK
   */
  readonly installLatestAwsSdk?: boolean;
```

I guess we should always have a @default tag for optional prop like this.

Reply:
Good point, will update it.